dev/bots/travis_script.sh

```diff
@@ -10,25 +10,43 @@ if [ "$SHARD" = "build_and_deploy_gallery" ]; then
   if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     echo "Building Flutter Gallery for Android..."
     export ANDROID_HOME=`pwd`/android-sdk
-    (cd examples/flutter_gallery; flutter build apk --release)
+    (
+      cd examples/flutter_gallery;
```

Comment:
I think you don't need the semicolons when it's split into multiple lines

Reply:
If you edit that example to:

```sh
#!/bin/bash
# GNU bash, version 4.3.46

(
  echo a
  echo b
  echo c 
)
```

Then it does the right thing.  A trailing backslash means "ignore this newline"